hnn_core/network_builder.py

```diff
@@ -503,11 +504,19 @@ def _connect_celltypes(self):
                         self.ncs[connection_name].append(nc)
 
     def _record_extracellular(self):
+        from .extracellular import _gather_nrn_voltages
```

Comment:
Is there a particular reason why this is imported locally within `_record_extracellular()`?

Reply:
My bad, I just noticed this was already brought up in a comment above.